en/documentation/index.md

```diff
@@ -71,6 +79,61 @@ The authoritative Ruby language documentation at [docs.ruby-lang.org][docs-rlo-e
 [RubyAPI.org][rubyapi-org]
 : Easily find and browse Ruby classes, modules, and methods.
 
+[ruby-doc.org][39]
+: online API documentation
+
+[DevDocs.io][40]
+: online API documentation
+
+[Ruby QuickRef][42]
+: The ruby quick reference
+
+[rubyreferences][43]
+    : A full language reference + detailed language changelog. Fantastic.
+
+### Style Guides
+
+[rubystyle.guide][44]
+: Rubocop's Ruby style guide
+
+[RuboCop][45]
+: Automated enforcement of their style guide.
+
+[Shopify][46]
+: Shopify's Ruby style guide
+
+[GitLab][47]
+: Gitlab's Ruby style guide
+
+[Airbnb][48]
+: AirBNB's Ruby style guide
+
+[w3resource][49]
```

Comment:
Thank you for your contribution and for adding helpful links 🙌🏻  I noticed this one has significant number of ads (screenshot below). While the content looks valuable, I wonder if we should consider whether this is a good addition to be on the documentation page on the Ruby official website 🤔 

![CleanShot 2024-11-25 at 21 51 56@2x](https://github.com/user-attachments/assets/ea19e436-a80b-482d-9316-b9bf3ce69ccc)